### PR TITLE
MAN-242 Include Updated Date in JSON response

### PR DESCRIPTION
**Pull request by nomadicoder**

### CODE REVIEW

File: app/serializers/blog_serializer.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class BlogSerializer
+class BlogSerializer < ApplicationSerializer
   include FastJsonapi::ObjectSerializer
```
Comment (bibliotechy) [inline app/serializers/blog_serializer.rb]: If `include FastJsonapi::ObjectSerializer` is in the super class, it doesn't need to be in all of the subclasses.

File: app/serializers/event_serializer.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class EventSerializer
+class EventSerializer < ApplicationSerializer # ~> NameError: uninitialized constant ApplicationSerializer
```
Comment (bibliotechy) [inline app/serializers/event_serializer.rb]: What's this error message?

File: app/schemas/blog_schema.json [hunk 3]
```diff
@@ -112,6 +112,16 @@
 								true,
                 false
 							]
+						},
+						"updated_at": {
+							"$id": "#/properties/data/properties/attributes/properties/updated_at",
+							"type": "string",
+							"title": "The time the blog record was updated",
+              "format": "date-time",
```
Comment (bibliotechy) [inline app/schemas/blog_schema.json]: what's up with the weird indentation?

File: app/serializers/event_serializer.rb (hunk 2 above)
Comment (nomadicoder) [reply app/serializers/event_serializer.rb]: @bibliotechy - That's from the Seeing Is Believing Vim plugin to execute ruby from within Vim.  That was generated by an errant keystroke. Please flag them if you see them. They should not be in the code.

**Pull request merged**